react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('value returned by useState:', isClicked)
+
+  function handleClick() {
+    console.log('value before calling setter:', isClicked)
+    setIsClicked(!isClicked);
+    console.log('value after calling setter:', isClicked)
+  }
+
+  return (
+    <button onClick={handleClick} style={isClicked ? { backgroundColor: color, fontSize:20, fontWeight: "bold" } : { backgroundColor: "white" }}>
```

Comment:
You could also pull this calculation into a local variable. That would make this button tag cleaner. 

You could also use ternary on just the backgroundColor, which would eliminate a tiny bit of duplication.